source/sass/blocks/page-header.scss

```
@@ -0,0 +1,12 @@
+.page-header {
+  background-color: $black-blue;
+  background-image: url("../img/back-mobile.png");
+  background-repeat: no-repeat;
+  margin: 0 auto;
+  min-height: 733px;
+  min-width: 320px;
+}
+
+.page-header-form {
```

Может так:
```suggestion
.page-header__form {
```
то есть класс элемента.